presto-main/src/test/java/com/facebook/presto/connector/MockConnectorFactory.java

```diff
@@ -93,16 +97,20 @@ public Connector create(String connectorId, Map<String, String> config, Connecto
         private final ConnectorContext context;
         private final Function<ConnectorSession, List<String>> listSchemaNames;
         private final BiFunction<ConnectorSession, String, List<SchemaTableName>> listTables;
+        private final BiFunction<ConnectorSession, SchemaTablePrefix, Map<SchemaTableName, ConnectorViewDefinition>> getViews;
         private final BiFunction<ConnectorSession, ConnectorTableHandle, Map<String, TpchColumnHandle>> getColumnHandles;
 
         public MockConnector(
```

Comment:
[nit] Probably it would be nice to have some `builder` here.

Reply:
Yep, makes sense. I will follow up in a separate PR.